profiles/common/modules/features/cce_basic_config/cce_basic_config.info

```diff
@@ -74,6 +71,9 @@ features[variable][] = image_captcha_fonts
 # The print_pdf_pdf_tool variable is set in cce_basic_config_strongarm_alter().
 # Please do not remove this entry when re-exporting the variables.
 features[variable][] = print_pdf_pdf_tool
+# The print_pdf_pdf_tool variable is set in cce_basic_config_strongarm_alter().
+# Please do not remove this entry when re-exporting the variables.
```

Comment:
That should be written in a README file because we regenerate the feature, we loose this kind of comment.